os/board/Kconfig

```diff
@@ -284,6 +284,13 @@ config BOARDCTL_RESET
 		Architecture specific logic must provide the board_reset()
 		interface.
 
+config BOARDCTL_RESET_HEADER
```

Comment:
I think reset might not be required, we can generalize it. Please consider modifying the name to generalize it, something like ```BOARDCTL_BOARD_HEADER``` or ```BOARDCTL_CHIP_HEADER```

Reply:
I checked. Thank you for your review.